components/error.tsx

```diff
@@ -1,14 +1,23 @@
 import { H3, Panel } from '@bigcommerce/big-design';
+import { ErrorMessageProps, ErrorProps } from '../types';
 
-interface ErrorMessageProps {
-    error?: Error;
-}
-
-const ErrorMessage = ({ error }: ErrorMessageProps) => (
-    <Panel>
+const ErrorContent = ({ message }: Pick<ErrorProps, 'message'>) => (
+    <>
         <H3>Failed to load</H3>
-        {error && error.message}
-    </Panel>
-);
+        {message}
+    </>
+)
+
+const ErrorMessage = ({ error, renderPanel = true }: ErrorMessageProps) => {
```

Comment:
we probably don't need to merge in all of these changes - remember, the steps are iterative and the main branch contains changes that are captured for other purposes (e.g. billing)